qiskit_ibm_runtime/accounts/account.py

```diff
@@ -20,15 +20,17 @@
 from typing_extensions import Literal
 
 from .exceptions import InvalidAccountError, CloudResourceNameResolutionError
-from ..api.auth import LegacyAuth, CloudAuth
+from ..api.auth import QuantumAuth, CloudAuth
+from ..channel import Channel
 from ..proxies import ProxyConfiguration
 from ..utils.hgp import from_instance_format
 from ..utils import resolve_crn
 
 AccountType = Optional[Literal["cloud", "legacy"]]
+ChannelType = Optional[Literal[Channel.IBM_CLOUD, Channel.IBM_QUANTUM]]
```

Comment:
Thanks, I was trying to not repeat strings everywhere but for some reason Enums and Literals don't play well. So I've decided to keep them as strings for now just like how it was originally since I want to release this soon.